spec/integrations/shared/base_render_examples.rb

```diff
@@ -91,6 +91,50 @@ def extract(field_name, object, _local_options, _options={})
     it('raises a BlueprinterError') { expect{subject}.to raise_error(Blueprinter::BlueprinterError) }
   end
 
+  context 'Given blueprint has ::field with a Proc :datetime_format argument' do
+    let(:result) do
+      '{"id":' + obj_id + ',"birthday":762739200,"deleted_at":null}'
+    end
+    let(:blueprint) do
+      Class.new(Blueprinter::Base) do
+        identifier :id
+        field :birthday,   datetime_format: -> datetime { datetime.strftime("%s").to_i }
+        field :deleted_at, datetime_format: -> datetime { datetime.strftime("%s").to_i }
+      end
+    end
+    it('returns json with a formatted field') { should eq(result) }
+  end
+
+  context 'Given blueprint has a Proc :datetime_format argument on an invalid ::field' do
+    let(:blueprint) do
+      Class.new(Blueprinter::Base) do
+        identifier :id
+        field :first_name, datetime_format: -> datetime { datetime.strftime("%s") }
+      end
+    end
+    it('raises a BlueprinterError') { expect{subject}.to raise_error(Blueprinter::BlueprinterError) }
```

Comment:
Hmm the error here is based on us using a datetime formatter on an invalid field, correct? The error message we get reads `'Cannot format DateTime object with invalid formatter'` The formatter in this test case is perfectly fine, no? I think we need an invalid field error or something.

Reply:
Nevermind I was confused. The message *DOES NOT* read `'Cannot format DateTime object with invalid formatter'`, rather it's `'Cannot format invalid DateTime object'` which is perfectly valid.